### PR TITLE
Set API version explicitly to v1alpha2 when condition is met.

### PR DESCRIPTION
**Pull request by fmuyassarov**
Set API version explicitly to v1alpha2 when v1alpha2 condition is met

### CODE REVIEW

**Comment by maelk** [comment]
Should we rather aim at using CentOS8, so that we don't rely on our own image ?

**Comment by fmuyassarov** [comment]
> Should we rather aim at using CentOS8, so that we don't rely on our own image ?

Yes, agree. But not sure if provisioning is going to work without any failures, but I'll test with the CentOS8 image that we have in CityCloud.

**Comment by fmuyassarov** [comment]
/hold 
testing with CentOS8

File: lib/images.sh [hunk 1]
```diff
@@ -11,8 +11,8 @@ elif [[ "${IMAGE_OS}" == "FCOS" ]]; then
   export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/30.20191014.0/x86_64/}
   export IMAGE_USERNAME=${IMAGE_USERNAME:-core}
 elif [[ "${IMAGE_OS}" == "Centos" ]]; then
-  export IMAGE_NAME=${IMAGE_NAME:-CentOS-7-x86_64-GenericCloud-1907.qcow2}
-  export IMAGE_LOCATION=${IMAGE_LOCATION:-http://cloud.centos.org/centos/7/images}
+  export IMAGE_NAME=${IMAGE_NAME:-centos-updated.qcow2}
+  export IMAGE_LOCATION=${IMAGE_LOCATION:-http://artifactory.nordix.org/artifactory/airship/images/centos.qcow2}
```
Comment (maelk) [inline lib/images.sh]: This needs to be changed to CentOS8

**Comment by maelk** [comment]
please update the description if we go with CentOS8

File: lib/images.sh (hunk 1 above)
Comment (fmuyassarov) [reply lib/images.sh]: yes, I'll for now I just want to make sure it is going to work with CentOS8

**Comment by fmuyassarov** [comment]
> please update the description if we go with CentOS8

sure, will do that

**Comment by alosadagrande** [comment]
Related issue #263 when creating CentOS 8 control-plane VM in the target cluster

**Comment by fmuyassarov** [comment]
> Related: issue #263 when creating CentOS 8 control-plane VM in the target cluster.
> 
> The idea is to replace centos-update.qcow2 (based in CentOS 7) with CentOS 8.

Thanks @alosadagrande . Can you then please open a PR once you finalize your tests

**Pull request merged**